libraries/standard/mqtt/include/types/iot_mqtt_types.h

```diff
@@ -998,7 +998,7 @@ typedef struct IotMqttNetworkInfo
              * interface when creating a new network connection. It is only valid when
              * #IotMqttNetworkInfo_t::createNetworkConnection is `true`.
              */
-            IotNetworkServerInfo_t * pNetworkServerInfo;
```

Comment:
Line 948-957 are not needed and should be removed.

Reply:
Good catch, removed.